app/responders/assign_editor_responder.rb

```diff
@@ -10,6 +10,8 @@ def define_listening
   end
 
   def process_message(message)
+    if:
+      title: "^Submit software|^Enviar software"
```

Comment:
This `if` option is a configuration setting, so it should be used in the configuration file, in the settings where the responder is declared, i.e.: https://github.com/mpadge/buffy/blob/ropensci/config/settings-production.yml#L54-L61

Reply:
So we would then have 2 configs, right? One as is, with the additional `if` clause, and then a second additional one for the other case. And then that other one would need an additional `responders/assign_stats_editor_responder.rb` file. Is that correct? In which case, modifying this PR to just insert those lines into the `config` would do, and i'll follow with a second PR to add the additional `if` clause and responder.

Reply:
Yes, that should work. 
I'd recommend to create a new PR with a fresh branch from ropensci and just one commit in the config file, to avoid future conflicts in other files.